CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## master
 
+## [v5.2.0][v5.2.0] (December 10, 2021)
+
+* Refactored gin middleware([#224](https://github.com/airbrake/gobrake/pull/224))
```

Comment:
I don't think this is worth mentioning in the changelog. Our users won't care about this change.

Reply:
I agree with Kyrylo here. If we mention anything here it would be to say, "Deprecating `NewMiddleware` func and replacing with `New`."

We actually missed something in https://github.com/airbrake/gobrake/pull/224/files. We should have refactored `NewMiddleware` to look like this:

```
func NewMiddleware(engine *gin.Engine, notifier *gobrake.Notifier) func(c *gin.Context) {
	return New(engine, notifier)
}
```